src/test/java/org/jenkinsci/plugins/workflow/log/LogStorageTestBase.java

```diff
@@ -350,12 +350,12 @@ protected String lines(CharSequence... lines) {
         return String.join(System.lineSeparator(), lines) + System.lineSeparator();
     }
 
-    private static class MockNode extends FlowNode {
+    protected static class MockNode extends FlowNode {
         MockNode(String id) {super(new MockFlowExecution(), id);}
         @Override protected String getTypeDisplayName() {return null;}
     }
 
-    private static class MockFlowExecution extends FlowExecution {
+    protected static class MockFlowExecution extends FlowExecution {
```

Comment:
(could be reverted)

Reply:
Reverted in https://github.com/jenkinsci/workflow-api-plugin/pull/296/commits/e92927515ac10b73ffa373b8bfdcf80ddedc5d3e.